devtools_m2/cypress/integration/CustomerCartInteractions/steps.js

```diff
@@ -39,6 +39,7 @@ When('I add a {string} widget to my cart', function(type) {
       break;
     case 'simple':
       // Do nothing
+      console.log('In simple case')
```

Comment:
Could we drop the console.log or convert it to https://docs.cypress.io/api/commands/log.html#Syntax ?